src/searching.md

```diff
@@ -183,148 +182,151 @@ Searches that ignore combining characters are slower than regular searches.
 Anki 2.1.24+, AnkiMobile 2.0.60+ and AnkiDroid 2.17+ support searching in notes with "regular expressions",
 a standard and powerful way of searching in text.
 
-Start a search with `re:` to search by regular expression. To make things easier, Anki will
+Start a search with `re:` to search using regular expression. To make things easier, Anki will
```

Comment:
If changing "by" to "using," the plural sounds more accurate. I can't explain why. 
`search using regular expressions.`

Reply:
```suggestion
Start a search with `re:` to search using regular expressions. To make things easier, Anki will
```

I agree with you.